vignettes/relase-process.qmd

```diff
@@ -0,0 +1,155 @@
+---
+title: "The Release Process"
+description: >
+  Reference for how to perform R package releases in the Hubverse
+vignette: >
+  %  \VignetteIndexEntry{The Release Process}
+  %  \VignetteEngine{quarto::html}
+  %  \VignetteEncoding{UTF-8}
+knitr:
+  opts_chunk: 
+    collapse: true
+    comment: '#>'
+---
+
+This document is adapted for The Hubverse from [The Carpentries Hubverse Developer's Handbook](https://carpentries.github.io/workbench-dev/releases.html) (c) The Carpentries. 
+
```

Comment:
I feel it would help to have 4 different and smaller sections with their own checklists that could help break down conceptually the general workflow.

1. First PR after a release: Describe the actions required, i.e. bump the version to a `devel` version, update the `NEWS.md` with a new `(development)` section etc.
2. Subsequent PRs. A few words about next PRs. Anything we need to be aware of?
3. Release PR: I think you've already covered this pretty well below
4.  Hotfixes on releases. Would be good to get this down on paper too so we know whet to do

Reply:
This structure would definitely make it more clear. I will implement it.